### PR TITLE
fix docker_api_version of repo

### LINKED ISSUE #333: Incorrect Docker API Version
When retrieving an existing docker respository with V2 enabled, there is a mismatch in Repository.docker_api_version and Repository.raw["dockerApiVersion"].

For example, create a new docker repository in the artifactory web console with V2 enabled. Then:

Retrieve the repository and enumerate the object's properties:
```python
repo = artifactory_.find_repository("james-test")
print("raw_docker_version", repo.raw["dockerApiVersion"])
print("object_property_docker_version", repo.docker_api_version)
```

You'll see something like this:
> raw_docker_version V2
> object_property_docker_version V1

If you update the repo using `repo.update()` the result is that `repo.docker_api_version` is used and the repository is incorrectly changed to V1.
https://github.com/devopshq/artifactory/blob/0.8.0/dohq_artifactory/admin.py#L771

I believe the culprit is the init function of RepositoryLocal sets the version to V1:
https://github.com/devopshq/artifactory/blob/0.8.0/dohq_artifactory/admin.py#L734

And then _read_response does not grab the real value from the API response:
https://github.com/devopshq/artifactory/blob/0.8.0/dohq_artifactory/admin.py#L791

So the default value of V1 is always used.

## Solution:
I propose the _read_response function should be changed to also update `self.docker_api_version` using the value from the api response.

### PR DESCRIPTION
**Pull request by donhui**
fix #333

### CODE REVIEW

File: dohq_artifactory/admin.py [hunk 1]
```diff
@@ -888,6 +889,7 @@ def _read_response(self, response):
         self.description = response.get("description")
         self.package_type = response.get("packageType")
         self._repositories = response.get("repositories")
+        self.docker_api_version = response.get("dockerApiVersion")
```
Comment (beliaev-maksim) [inline dohq_artifactory/admin.py:892]:
```suggestion
        self.docker_api_version = response.get("dockerApiVersion", None)
```

File: dohq_artifactory/admin.py [hunk 2]
```diff
@@ -1006,6 +1008,7 @@ def _read_response(self, response):
         self.repo_layout_ref = response.get("repoLayoutRef")
         self.archive_browsing_enabled = response.get("archiveBrowsingEnabled")
         self.url = response.get("url")
+        self.docker_api_version = response.get("dockerApiVersion")
```
Comment (beliaev-maksim) [inline dohq_artifactory/admin.py:1011]:
```suggestion
        self.docker_api_version = response.get("dockerApiVersion", None)
```

**Review by beliaev-maksim — CHANGES_REQUESTED** [review]
if I am not mistaken, then this feature is not available in all versions of artifactory. Then, if user uses some old version it may get broken. Adding `None` should help

File: dohq_artifactory/admin.py [hunk 3]
```diff
@@ -805,6 +805,7 @@ def _read_response(self, response):
         self.package_type = response.get("packageType")
         self.repo_layout_ref = response.get("repoLayoutRef")
         self.archive_browsing_enabled = response.get("archiveBrowsingEnabled")
+        self.docker_api_version = response.get("dockerApiVersion")
```
Comment (donhui) [inline dohq_artifactory/admin.py:808]:
```suggestion
        self.docker_api_version = response.get("dockerApiVersion")
```

**Comment by donhui** [comment]
@beliaev-maksim
Thanks for your suggestion, I hava updated the code.

**Review by beliaev-maksim — APPROVED** [review]

**Review by allburov — APPROVED** [review]

**Pull request merged**